.github/workflows/releaser.yaml

```diff
@@ -52,5 +52,5 @@ jobs:
         with:
           tag_name: ${{ github.ref }}
           release_name: ${{ github.ref }}
-          draft: false
+          draft: true
```

Comment:
what is this changed for?

Reply:
by default the release has no good description but is sent out with email nevertheless. better to start as draft, manually update the description and then release